src/main/kotlin/snyk/container/YAMLImageExtractor.kt

```diff
@@ -5,10 +5,25 @@ import com.intellij.openapi.project.Project
 import com.intellij.openapi.vfs.VirtualFile
 import com.intellij.psi.PsiFile
 import io.snyk.plugin.findPsiFileIgnoringExceptions
+import org.intellij.lang.annotations.Language
 
 object YAMLImageExtractor {
     private val logger = Logger.getInstance(javaClass)
-    private val imageRegEx = "(?<=([-\\s]image:))(\\s*[a-zA-Z0-9./\\-_]+)(:)?([a-zA-Z0-9./\\-_]+)?".toRegex()
+
+    // see https://kubernetes.io/docs/concepts/containers/images/
+    @Language("regexp")
```

Comment:
@ArtsiomCh , why do we need this `@Language` annotation?

Reply:
to see regex highlighting in Idea (language injection)

Reply:
got it, thanks.